MetaMask/metamask-extension
# Add network constants for ETC chains

## **Description**

This PR adds "Ethereum Classic" in the network list as a popular custom network.

## **Related issues**

None

## **Manual testing steps**

1. Go to Add Networks
1. Check that "Ethereum Classic" is there.

## **Screenshots/Recordings**

### **Before**

<img width="929" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/802700/b114c588-c888-4612-bb31-1851776603a4">

### **After**

<img width="928" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/802700/6203a2a5-2337-422b-a52c-439df405d40a">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
